src/components/team/TeamCard.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from "next/link";
+import { Router, useRouter } from "next/router";
+import React from "react";
+import { TeamCardI } from "./team-interfaces";
+
+const TeamCard: React.FC<TeamCardI> = ({
+	image,
+	name,
+	position,
+	description,
+	socials,
+	email,
+	phone,
+	address,
+}) => {
+	const router = useRouter();
+	const { facebook_url, twitter_url } = socials;
+	// function sends targeted data to targetpath using nextRouter
+	function sendSingleTeamProp() {
+		router.push({
+			pathname: "/team/single-team",
+			query: {
+				image,
+				name,
+				position,
+				description,
+				email,
+				phone,
+				address,
+				facebook_url,
+				twitter_url,
+			},
+		});
+	}
+
+	return (
+		<div className="team-card">
+			<div className="team-card-img">
+				<img src={image} alt="" />
+			</div>
+			<div className="team-card-desc">
+				<h5 onClick={sendSingleTeamProp}>
```

Comment:
```tsx
<h5 onClick={sendSingleTeamProp}>
					<a className="link-dark text-decoration-none">{name}</a>
```
this should be 

```tsx
	<h5>
					<Link href={`/team/${team?.id}`}>
						<a className="link-dark text-decoration-none">{name}</a>
					</Link>
				</h5>
``

Reply:
The single team page expects all props data from team card component.

Without the function getting triggered, how does it get achieved?

Outside loading up an available dynamic route, what does this do please?  ```  href={`/team/${team?.id}`} ```